debian/wazo-dist-upgrade.install

```diff
@@ -0,0 +1,7 @@
+bin/wazo-dist-upgrade usr/bin/
+bin/wazo-dist-upgrade-buster-check usr/lib/wazo-dist-upgrade/
+bin/wazo-dist-upgrade-buster usr/bin/
+bin/wazo-dist-upgrade-bullseye usr/bin/
```

Comment:
`bin/wazo-dist-upgrade-bullseye` does seem to exist.
And `-base` and `-ce` seem to be expected in `/usr/bin/` instead of `usr/lib/dist-upgrade`
```suggestion
bin/wazo-dist-upgrade-bullseye-base usr/bin/
bin/wazo-dist-upgrade-bullseye-ce usr/bin/
```